extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -89,6 +100,27 @@ export class AmpExperiment extends AMP.BaseElement {
       return experiments;
     });
   }
+
+  /**
+   * Parses URL fragment and returns variant override.
+   * @return {!Object<string, string>}
+   * @private
+   */
+  getVariantOverride_() {
+    const queries = parseQueryString(
```

Comment:
Should we review which parameters make their way there? Some of those parameters could be relatively sensitive.